src/Http/Converter/RequestUriToSql.php

```diff
@@ -935,21 +935,42 @@ public static function parseRelationShips(string $relationships, &$results) : ar
         return $newResults;
     }
 
+    /**
+     * Get table columns.
+     */
+    public function getTableColumns() : array
+    {
+        $fields = $this->model->getReadConnection()->describeColumns($this->model->getSource());
+        $columns = [];
+
+        foreach ($fields as $field) {
+            $columns[$field->getName()] = $field->getName();
+        }
+
+        return $columns;
+    }
+
     /**
      * Set CustomSort for the query.
      *
      * @param string $sort
      *
-     * @return string
+     * @return void
      */
     public function setCustomSort(?string $sort) : void
     {
-        if (!is_null($sort)) {
+        if (!is_null($sort) && Str::contains($sort, '|')) {
```

Comment:
This will only allow to sort when there's a `|` on the string. Sorting without specifying the order wouldn't work, e.g. `?sort=name`